spinalcordtoolbox/scripts/sct_compute_snr.py

```diff
@@ -185,6 +198,13 @@ def main(argv=None):
     # Display result
     if fname_mask:
         printv('\nSNR_' + method + ' = ' + str(snr_roi) + '\n', type='info')
+    
+    #Added function for text file
+    if file_name is not None:
+        f = open(file_name,"w")
+        f.write(str(snr_roi))
+        f.close
+        printv('\nFile saved to '+ file_name)
```

Comment:
This pattern can be simplified by using Python's [`with` statement](https://realpython.com/python-with-statement/):

```suggestion
        with open(file_name, "w") as f:
            f.write(str(snr_roi))
            printv('\nFile saved to ' + file_name)
```

That way, you don't need to write `f.close`.